src/httpserver.cpp

```diff
@@ -614,7 +619,7 @@ CService HTTPRequest::GetPeer()
         const char* address = "";
         uint16_t port = 0;
         evhttp_connection_get_peer(con, (char**)&address, &port);
-        peer = CService(address, port);
+        LookupNumeric(address, peer, port);
```

Comment:
You ignore the return value of `Lookup*` almost everywhere. Is this on purpose?
I think the old behavior was to return a service with `!IsValid()` when the lookup fails. But if that is the case we could just as well have the API be:
```
peer = LookupNumeric(address, port);
```